docs/release-notes/cloud/cloud-2021-11-02-legacy-organizations.md

```diff
@@ -19,7 +19,7 @@ Synced organizations have several advantages, such as keeping the repositories a
 
 However, the following scenarios may impact you:
 
--   **If your legacy organization is using the Open source plan** then Codacy has already stopped analyzing your repositories. To restore the analysis of your repositories you can wait until Codacy attempts to migrate your organization, or restore the analysis immediately by [performing the migration yourself](../../faq/repositories/how-do-i-migrate-a-legacy-repository-to-a-synced-organization.md).
+-   **If your legacy organization is using the Open source plan** then Codacy has already stopped analyzing your repositories. To restore the analysis of your repositories please wait until Codacy attempts to migrate your organization.
```

Comment:
```suggestion
-   **If your legacy organization is using the Open source plan** then Codacy has already stopped analyzing your repositories.
```
Sounds it is unnecessary, if feels like an action, but it isn't an action. 

Or we could do something like this:
```suggestion
-   **If your legacy organization is using the Open source plan** then Codacy has already stopped analyzing your repositories. Codacy will attempt to migrate your organization and restore the analysis of your repositories.
```

Reply:
Yes, I like your second proposal best! Thank you @inesgomas.